.github/workflows/pr-intake.yml

```diff
@@ -0,0 +1,157 @@
+name: pr-intake
+
+on:
+    pull_request:
+        branches: 
+        - main
+        - 'releases/**'     
+        paths:
+        - 'Examples/**'
+        - 'SemiconductorTestLibrary.Abstractions/**'
+        - 'SemiconductorTestLibrary.Extensions/**'
+        - 'SemiconductorTestLibrary.TestStandSteps/**'
+        - 'TestAssets/**'
+        - '.github/**'
+
+env:
+    GitHub_Repo_Name: "semi-test-library-dotnet"
+    PR_Number: "#${{ github.event.number }}"
+
+jobs:
+  wait-for-pr-build-and-test-workflows:
+    runs-on: ubuntu-latest
+    environment: gh-action-testing
+    steps:
+      # Before fetching the run ID, we need to wait for the PR Build and PR Test Workflow run to start as it uses in_progress type of workflow_run event.
+      - name: Wait for PR Build and PR Test Workflow to Start
+        run: sleep 10
```

Comment:
@Tanish-NI. are we sure this is enough time? Maybe 30 seconds is safer?

Reply:
Yes, 30 seconds sounds safer to assume that pr-build and pr-test runs should be triggered.
Updated!